src/status_im/contexts/chat/messenger/composer/style.cljs

```diff
@@ -10,13 +10,12 @@
 (def border-top-radius 20)
 
 (defn shadow
-  [focused? theme]
-  (if platform/ios?
+  [theme]
+  (when platform/ios?
     {:shadow-radius  20
      :shadow-opacity (colors/theme-colors 0.1 0.7 theme)
      :shadow-color   colors/neutral-100
-     :shadow-offset  {:width 0 :height (colors/theme-colors -4 -8 theme)}}
-    {:elevation (if @focused? 10 0)}))
+     :shadow-offset  {:width 0 :height (colors/theme-colors -4 -8 theme)}}))
```

Comment:
Is the elevation here not needed? Does the shadow look right with all combinations of being focused and not focused, and having content and not having content?

Reply:
Yes, it looks fine. But just to be double sure, @pavloburykh Please keep an eye on this while testing.

Reply:
I have checked those cases, shadow looks okay to me